src/main/java/seedu/address/model/person/User.java

```diff
@@ -21,15 +19,14 @@ public class User extends Person {
     protected User(Name name, Phone phone, Email email, Address address,
                    TelegramHandle telegramHandle, Set<GroupTag> groupTags, Set<ModuleTag> moduleTags) {
         super(name, phone, email, address, telegramHandle, groupTags, moduleTags);
-        requireAllNonNull(telegramHandle, moduleTags);
     }
 
     public static User getSingletonUser(Name name, Phone phone, Email email, Address address,
         TelegramHandle telegramHandle, Set<GroupTag> groupTags, Set<ModuleTag> moduleTags) {
 
-        if (user == null) {
-            user = new User(name, phone, email, address, telegramHandle, groupTags, moduleTags);
-        }
+        user = new User(name, phone, email, address, telegramHandle, groupTags, moduleTags);
+        System.out.println(user);
```

Comment:
Please remember to remove the debug statement.

Reply:
Fixed.